contracts/base/NFTDrop.sol

```diff
@@ -0,0 +1,307 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+//  ==========  External imports    ==========
+
+import "@openzeppelin/contracts/utils/Multicall.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
```

Comment:
can we use our own 'copies' of these? I added `lib/TWStrings.sol` and `feature/Multicall.sol` that are the same as these. 

goal is to have ppl use our base without having to install openzeppelin if possible